controllers/openstack_ansibleee_controller.go

```diff
@@ -55,10 +56,14 @@ const (
 type OpenStackAnsibleEEReconciler struct {
 	client.Client
 	Kclient kubernetes.Interface
-	Log     logr.Logger
```

Comment:
That change is intentional and global,
it came up in a discussion with Gibi: 
https://github.com/openstack-k8s-operators/keystone-operator/pull/220#issuecomment-1483117420

the purpose of that is to prepare a way to avoid a race condition in case we run reconciles in parrallel, as each reconcile id would be affected by the context ctx objet, so I don't want to share the ctx between all reconciles beceause we wouldn't know what recncile id is the acutal in parallel reconcile runs.